src/components/ChatTile.vue

```diff
@@ -57,20 +63,35 @@ export default {
       chat,
       send,
       text: "",
+      model: null,
+      displayWarning: false,
     };
   },
+  beforeCreate() {
+    const threshold = 0.9;
+    load(threshold).then(model => {
+      this.model = Object.freeze(model);
```

Comment:
I can't reproduce this issue so I am not sure how to go about this.
The model is always present even before the loading spinner disappears in my local dev environment.
As for the reason why it is running twice, I don't think it has anything to do with the model. I tried adding a `beforeCreate` hook on the main branch and it is running twice. Maybe it has something to do with how the components are setup?

Reply:
I also don't think we can start the loading process any time earlier than in the `beforeCreate` hook. When I was building the demo, I considered whether to add it before there or in the `beforeMount` hook and decided on `beforeCreate` since it is the earliest hook in the lifecycle of the ChatTile component.
Unless you want to move it higher in the component hierarchy which I don't think is ideal because I believe it makes more sense to keep all chat input management related logic within the ChatTile component.

Reply:
Alright, let's leave the double-call! 

Re the exception and model load: I think it makes sense that this issue is intermittent, since it seems like a timing issue with how fast the model load takes place. For example, locally I get this exception below - the model does not load until about 20-30 seconds into the call, and if I try to send a chat message the logic uses a model that is still `null`: 
<img width="513" alt="image" src="https://user-images.githubusercontent.com/65890040/235855319-b89ca5f2-f67b-4123-8529-c54d80fe60ca.png">

Which makes sense, since from what I understand we don't actually await for the model load to finish anywhere before proceeding. I've got some ideas to possibly mitigate this, will file a PR against this branch for your review in a bit!